apps/nova/src/components/Chat/Messages.tsx

```diff
@@ -270,9 +273,13 @@ const Messages: React.FC<{
               </ul>
             )}
           </div>
-
-          <div className="px-4 pt-2 pb-6">
-            <SendInput onSubmit={submit} />
+          {isTyping ? (
+            <div className="px-6">
+              <UserTyping {...otherMember} imageUrl={otherMember.image} />
+            </div>
+          ) : null}
+          <div className="px-4 pt-2 pb-6 mt-3">
+            <SendInput typeMessage={typingMessage} onSubmit={submit} />
```

Comment:
Let's give attention to naming and make it consistent.

```suggestion
            <SendInput typingMessage={typingMessage} onSubmit={submit} />
```